.github/workflows/ci.yml

```diff
@@ -291,6 +286,16 @@ jobs:
           yarn rw g page ciTest
         working-directory: ${{ steps.setup_test_project.outputs.test_project_path }}
 
+      - name: Run "g sdl"
+        run: |
+          yarn rw g sdl userExample
+        working-directory: ${{ steps.setup_test_project.outputs.test_project_path }}
+
+      - name: Run "rw type-check"
+        run: |
+          yarn rw type-check
+        working-directory: ${{ steps.setup_test_project.outputs.test_project_path }}
+
```

Comment:
Depending on how thorough you want to be here, you could switch strict to be true in the tsconfig to _really_ ensure things are valid for all consumers

Reply:
Hmmm good point! I don’t think I’ve ever tried it in strict mode 😂